cmd/test-tool/README.md

```diff
@@ -0,0 +1,58 @@
+
+When using the test tool to batch-send test transactions, you can dynamically adjust the number of test objects and the
+transaction sending rate in the configuration file to control the test volume. This allows for routine automated testing
+and stress testing of the Exocore chain.
+
+## functionalities
+
+The current implementation primarily provides the following functionalities:
+
+1. Creates the required test objects, including assets, stakers, operators, and AVS, based on parameters specified in
+   the configuration file, and saves them to the local SQLite database.
+2. Transfers a specified amount of Exo tokens to the created test objects, the amount has been defined in the
+   configuration file, which will be used to sign subsequent test transactions.
+3. Registers the created test objects onto the Exocore chain.
+4. Saves the existing assets and AVS (dogfood) information on the Exocore chain to the local SQLite database.
+5. Adds the newly created test assets to the asset list supported by the dogfood AVS.
+6. Opts in the newly created test operators into all test-created AVS and the dogfood AVS.
+7. Uses the private keys of all created stakers to batch-send deposit test transactions. Each asset initiates a test
+   deposit transaction. After sending, the tool verifies the on-chain status and asset state of all transactions
+   according to the configuration file parameters. The relevant transaction information and verification results are
+   saved to the local SQLite database.
+8. Uses the private keys of all stakers to batch-send delegation transactions. Each staker delegates their deposited
+   assets equally to all operators. After sending, the tool verifies the on-chain status and asset state of all
+   transactions according to the configuration file parameters. The relevant transaction information and verification
+   results are saved to the local SQLite database.
+9. Uses the private keys of all stakers to batch-send undelegation transactions. Each staker undelegates all delegations
+   performed in the previous step. After sending, the tool verifies the on-chain status and asset state of all
+   transactions according to the configuration file parameters. The relevant transaction information and verification
+   results are saved to the local SQLite database.
+10. Uses the private keys of all created stakers to batch-send withdrawal test transactions. Each asset initiates a test
+    withdrawal transaction. After sending, the tool verifies the on-chain status and asset state of all transactions
+    according to the configuration file parameters. The relevant transaction information and verification results are
+    saved to the local SQLite database.
+11. After waiting for a specified period as defined in the configuration file, the tool repeatedly performs the tests
+    described in steps 7, 8, 9, and 10 in a loop.
+
+## commands
+
+* The `init` command can be used to create a default configuration file.
```

Comment:
it's better to add the  whole command samples into the doc. so we can directly see the command clearly.

### Step 1: Initialize in current directory
test-tool init --home .

### Step 2: Prepare test environment
test-tool prepare --home .

### Step 3: Start the test tool
test-tool start --home .

### Step 4: Run batch tests
test-tool batch-test depositLST --home .

### Query transactions for specific batch and status
test-tool query-tx-record <msgType> <batch-id> <status>

Reply:
it will be added in the next commit.